src/init.cpp

```diff
@@ -644,6 +645,9 @@ void ThreadImport(std::vector<boost::filesystem::path> vImportFiles)
         LogPrintf("Stopping after block import\n");
         StartShutdown();
     }
+
+    LoadMempool();
+    ScheduleDumpMempool(*scheduler);
```

Comment:
If we're going to dereference scheduler unconditionally, it should be passed as a reference rather than a pointer.